src/broadcast.rs

```diff
@@ -70,61 +70,64 @@
 //! // Create a random set of keys for testing.
 //! let netinfos = NetworkInfo::generate_map(0..NUM_NODES);
 //!
-//! // Create initial nodes by instantiating a `Broadcast` for each:
-//! let mut nodes: BTreeMap<_, _> = netinfos.into_iter().map(|(i, netinfo)| {
-//!     let bc = Broadcast::new(Arc::new(netinfo), PROPOSER_ID)
-//!                  .expect("could not instantiate Broadcast");
-//!
-//!     (i, bc)
-//! }).collect();
+//! // Create initial nodes by instantiating a `Broadcast` for each.
+//! let mut nodes: BTreeMap<_, _> = netinfos
+//!     .into_iter()
+//!     .map(|(i, netinfo)| {
+//!         let bc =
+//!             Broadcast::new(Arc::new(netinfo), PROPOSER_ID).expect("Broadcast instantiation");
```

Comment:
That's why we use `rustfmt`. The formating just "is" :) I wouldn't know how to format otherwise because the number of possibilities is too large. `Broadcast` was capitalized in the previous version. It's just the name of the type. I think it's better to keep it capitalized for that reason.